kie-spring-boot/kie-spring-boot-autoconfiguration/jbpm-spring-boot-autoconfiguration/pom.xml

```diff
@@ -73,6 +73,12 @@
     <dependency>
       <groupId>org.jbpm</groupId>
       <artifactId>jbpm-kie-services</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>javax.validation</groupId>
+          <artifactId>validation-api</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
this is brought by kie-soup-dataset-api and it has 1.0 version while spring boot web requires 2.1 so excluding it here as it will be added by spring anyway